packages/eslint-config-datacamp/index.js

```diff
@@ -72,7 +80,26 @@ module.exports = {
     'import/prefer-default-export': 'off',
     'jest/no-deprecated-functions': 'off', // Needs to know the jest version, not possible from a shared config
     'max-classes-per-file': 'off',
+    'node/callback-return': 'warn',
```

Comment:
It's not something we've discussed in the meeting yet but in general, we try to set it either `off` or `error`. Warnings should be mostly used for transition periods IMO. Nobody looks at warnings, you only see it in the editor. With current tooling, it's very easy to turn off a rule when it's not relevant and if you ignore it half of the time anyway, I don't think the rule provides a lot of value.

So to me, these should be either set to "off" or to "error" but never "warn"